<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestione Membri Squadra</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_Main">
   <item>
    <widget class="QLabel" name="lblTitoloGestioneMembri">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Gestione Membri Squadra</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblNomeSquadra">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Squadra: [Nome Squadra]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="lstMembriSquadra"/>
   </item>
   <item>
    <widget class="QWidget" name="aggiungiMembroContainer" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_AggiungiMembro">
      <item>
       <widget class="QLineEdit" name="txtNuovoMembro">
        <property name="placeholderText">
         <string>Nome nuovo membro...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAggiungiMembro">
        <property name="text">
         <string>Aggiungi Membro</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnRimuoviMembro">
     <property name="text">
      <string>Rimuovi Membro Selezionato</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>